readthedocs/builds/models.py

```diff
@@ -440,6 +440,11 @@ def post_save(self, was_active=False):
             trigger_build(project=self.project, version=self)
         # Purge the cache from the CDN.
         version_changed.send(sender=self.__class__, version=self)
+    
+    def recreate(self):
+        self.clean_resources()
+        trigger_build(project=self.project, version=self)
+        version_changed.send(sender=self.__class__, version=self)
```

Comment:
Will this not trigger on the `save()` above? I guess this is because we're removing assets after the save?